src/webgl/shaders/filters/blur.frag

```diff
@@ -0,0 +1,50 @@
+// Single-pass blur filter, taken from Adam Ferriss' repo of shader examples:
```

Comment:
How does the output of this blur compare to the cpu one?

Reply:
Here's a sketch for comparison, although bricks.jpg isn't the best example to compare blur:
https://editor.p5js.org/jwong/sketches/r9fxPerPU